tablo/models.py

```diff
@@ -745,6 +770,11 @@ def update_feature(self, feature):
                         argument_values.append(feature['attributes'][key])
                     else:
                         argument_values.append(datetime.fromtimestamp(feature['attributes'][key] / 1000))
+                elif key in image_fields and feature['attributes'][key]:
+                    image_path = FeatureServiceLayer.create_image_path(self.service.id, "NO_PRIMARY_KEY", key)
+                    img_base64_data = FeatureServiceLayer.process_image_data(feature['attributes'][key], image_path,
```

Comment:
Like this:
                   img_base64_data = FeatureServiceLayer.process_image_data(
                        feature['attributes'][key], image_path, images_large, images_thumbs
                   )